src/main/java/com/sublinks/sublinksapi/comment/repositories/CommentRepositoryImpl.java

```diff
@@ -54,17 +59,38 @@ public List<Comment> allCommentsBySearchCriteria(CommentSearchCriteria commentSe
     if (commentSearchCriteria.post() != null) {
       predicates.add(cb.equal(commentTable.get("post"), commentSearchCriteria.post()));
     }
+    // @todo: implement comment read
     // Join for CommentView
-    if (commentSearchCriteria.person() != null) {
-      final Join<Comment, CommentRead> commentReadJoin = commentTable.join("commentReads",
-          JoinType.LEFT);
-      commentReadJoin.on(cb.equal(commentReadJoin.get("person"), commentSearchCriteria.person()));
-    }
+//    if (commentSearchCriteria.person() != null) {
+//      final Join<Comment, CommentRead> commentReadJoin = commentTable.join("commentReads",
+//          JoinType.LEFT);
+//      commentReadJoin.on(cb.equal(commentReadJoin.get("person"), commentSearchCriteria.person()));
+//    }
```

Comment:
Did you intend to leave this commented out code?

Reply:
I can remove it.